### PR TITLE
Add Q-Gears (Final Fantasy VII)

### PR DESCRIPTION
**Pull request by pcwalton**

### CODE REVIEW

File: games/q.yaml [hunk 1]
```diff
@@ -1,3 +1,17 @@
+- name: Q-Gears
+  images:
+  - http://q-gears.sourceforge.net/ss.jpg
+  lang: C++
+  license:
+  - GPL2
+  development: halted
+  originals:
+  - Final Fantasy VII
```
Comment (cxong) [inline games/q.yaml]: This game will need to be added in the originals

File: games/q.yaml [hunk 2]
```diff
@@ -1,3 +1,17 @@
+- name: Q-Gears
+  images:
+  - http://q-gears.sourceforge.net/ss.jpg
+  lang: C++
```
Comment (cxong) [inline games/q.yaml]:
```suggestion
  lang: C++
  framework: OGRE
```

File: games/q.yaml [hunk 3]
```diff
@@ -1,3 +1,17 @@
...
+  license:
+  - GPL2
```
Comment (cxong) [inline games/q.yaml]:
```suggestion
  - GPL2
content: commercial
```

File: games/q.yaml (hunk 1 above)
Comment (wonea) [reply games/q.yaml]: Sorted

**Pull request merged**